emission/tests/analysisTests/modeinferTests/TestPipeline.py

```diff
@@ -29,7 +35,7 @@ def setUp(self):
     self.assertEquals(self.SectionsColl.find().count(), 0)
 
     etc.loadTable(self.serverName, "Stage_Modes", "emission/tests/data/modes.json")
-    etc.loadTable(self.serverName, "Stage_Sections", "emission/tests/data/testModeInferFile")
+    etc.loadTable(self.serverName, "Stage_Sections", "emission/tests/data/newTestModeInferFilePlacesDeleted")
```

Comment:
I still think this won't work. You don't want to load the new-style data into `Stage_Sections`